ezpublish/EzPublishKernel.php

```diff
@@ -67,7 +66,6 @@ public function registerBundles()
             new LiipImagineBundle(),
             new FOSHttpCacheBundle(),
             new EzPublishCoreBundle(),
-            new EzPublishLegacyBundle( $this ),
```

Comment:
As long as legacy-bridge is in require, we can just as well keep this for now saving us from having to change prepare_ezpublish.sh.